### PR TITLE
Formatting doc

### PR DESCRIPTION
**Pull request by acucunato**
- updated API to fix gender search issue
- added new results photo to read me
- updated JS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ Pooch Quest is an application that assists the user in finding their perfect dog
 
 ## Contributers
 
-- **Rana Fayez** - _HTML, CSS, Materialize, Jquery_ - [Rana Fayez](https://github.com/tagine "Rana Fayez")
-- **Alicia Burn** - _HTML, CSS, Materialize_ - [Alicia Burn](https://github.com/AliciaBurn "Alicia Burn")
-- **Alyssa Cucunato** - _HTML, CSS, Jquery, Javascript, AJAX Requests, APIs_ - [Alyssa Cucunato](https://github.com/acucunato/pooch-quest "Alyssa Cucunato")
-- **Jordan Smith** - _HTML, CSS, Jquery, Javascript, AJAX Requests, APIs_ -[Jordan Smith](https://github.com/jsmithxyz "Jordan Smith")
+- **Rana Fayez** - *HTML, CSS, Materialize, Jquery* - [Rana Fayez](https://github.com/tagine "Rana Fayez")
```
Comment (acucunato) [inline README.md:53]: just updated all of us so it italicizes in our readme

File: script.js [hunk 2]
```diff
@@ -71,7 +72,10 @@ $(document).ready(function() {
                   "A few treasures about this pooch: " +
                   "</p>" +
                   "<p>" +
-                  description +
+                  description + " " +
```
Comment (acucunato) [inline script.js:75]: added URL link to end of description, user can click link - looks more polished

File: script.js [hunk 3]
```diff
@@ -23,7 +24,7 @@ $(document).ready(function() {
             breed +
             "&location=" +
             location +
-            "&sex=" +
+            "&gender=" +
```
Comment (acucunato) [inline script.js:27]: *debugged* updated API now it filters by gender

File: script.js [hunk 4]
```diff
@@ -216,7 +220,7 @@ $(document).ready(function() {
 
 //Event Listener for Fairy Dust
 $(".sidenav-trigger.waves-effect.waves-light.btn-large").click(function(event) {
-  console.log("Hitting sidebar");
+
```
Comment (acucunato) [inline script.js:223]: removed console.log to create clean code

**Review by jsmithxyz — APPROVED** [review]

**Pull request merged**